index.js

```diff
@@ -4,18 +4,22 @@ var redis = Redis.createClient({host: process.env.REDIS_URL || 'redis'});
 var express = require('express');
 var cors = require('cors');
 var app = express();
+var since = redis.get('npm:latest_seq') || 'now'
 
 var changes = new ChangesStream({
   db: 'https://replicate.npmjs.com/registry',
   include_docs: true,
-  since: 'now'
+  since: since
 });
 
 changes.on('data', function (change) {
   var name = change.doc.name
+  var version = change.doc['dist-tags'].latest
   if(name){
-    console.log(name)
+    console.log(change.seq, name, version)
     redis.lpush('npm-updated-names', name)
+    redis.lpush('npm-updated-names-with-versions', name + ' ' + version)
```

Comment:
maybe push an obj? `{name: name, version: version}`

Reply:
Ooh how bout push the seq here too? Then we can just lrange 0 0 on load?
`{name: name, version: version, seq: change.seq}`

Reply:
Quick update: I've been trying to make this work but the npm redis lib [is totally async](https://www.npmjs.com/package/redis), and doesn't support Promises yet. Trying to wrap everything in a callback currently.